interpreter/text/parser.mly

```diff
@@ -730,13 +772,13 @@ module_fields1 :
       { m with funcs = funcs @ m.funcs;
         imports = ims @ m.imports; exports = exs @ m.exports } }
   | elem module_fields
-    { fun c -> let mf = $2 c in
-      fun () -> let m = mf () in
-      {m with elems = $1 c :: m.elems} }
+    { fun c -> let ef = $1 c in let mf = $2 c in
```

Comment:
Guessing you had a lot of fun figuring out the right phasing here. :) Sorry for the mess.